dwave/embedding/transforms.py

```diff
@@ -356,7 +356,7 @@ def embed_bqm(source_bqm, embedding=None, target_adjacency=None,
         >>> embedding = {'a': {0}, 'b': {1}, 'c': {2, 3}}
         >>> # Embed and show the chain strength
         >>> target_bqm = dwave.embedding.embed_bqm(bqm, embedding, target)
-        >>> target_bqm.quadratic[(2, 3)]
+        >>> print(target_bqm.quadratic[(2, 3)])
```

Comment:
I think we can use [ELLIPSIS](https://www.sphinx-doc.org/en/master/usage/extensions/doctest.html) here